contract/src/vendors/update_vendors.rs

```diff
@@ -33,7 +33,7 @@ impl Contract {
 
     /// Adds a list of items to a specific vendor's store-front
     pub fn add_item_to_vendor(&mut self, vendor_id: AccountId, items: Vec<ExtVendorItem>) {
-        self.assert_admin_or_vendor(&vendor_id);
+        self.assert_vendor();
```

Comment:
will vendors have a portal for updating their vendor metadata or items? allowing admins to change this could reduce scope of work and simplify process. 

Ex. vendor can just use a google form to request changes, and then admins can make changes through CLI or admin page or something.

Reply:
goes for updating and adding items, and updating metadata

Reply:
out of scope for this year but can consider it for future conferences.